arch/ARM/ARMMapping.c

```diff
@@ -887,6 +887,7 @@ static void add_cs_detail_general(MCInst *MI, arm_op_group op_group,
 			ARM_get_detail_op(MI, 2)->shift.type = ARM_SFT_LSL;
 			ARM_get_detail_op(MI, 2)->shift.value = ShAmt;
 		}
+		ARM_inc_op_count(MI);
```

Comment:
This is done via `ARM_set_mem_access(MI, false);`

Reply:
I remember testing the ARM instruction `ldrsht` and the memory operand was missing from the operands list. So I tried debugging it and it seemed like the operand counter wasn't incremented. I don't know if it was supposed to be incremented somewhere else but apparently it didn't

Reply:
Is also fixed by #1949. So you just have to resolve the conflict during the future rebase.